cobigen-templates/templates-oasp4j/src/main/templates/constants/security_permissions/templates.xml

```diff
@@ -1,11 +1,15 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<templatesConfiguration xmlns="http://capgemini.com/devonfw/cobigen/TemplatesConfiguration" version="2.1">
-    <templates>
-        <template name="permission_constants" destinationPath="src/main/java/${variables.rootPackage}/general/common/api/constants/PermissionConstants.java" templateFile="PermissionConstants.java.ftl"/>
-    </templates>
-    <increments>
-        <increment name="permission_constants" description="Permission Constants">
-            <templateRef ref="permission_constants"/>
-        </increment>
-    </increments>
-</templatesConfiguration>
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>

```

Comment:
The formatting is odd. Please edit xml and java files just in eclipse. Nowhere else.